### ISSUE TITLE
Can I share to specify chat?

### ISSUE THREAD

**Issue by niuhuan** [ShoutSocial/share_handler#95]
I want to select some dialog boxes before sharing, which makes sharing more convenient. Just like the picture, I took screenshots of iOS and Android separately

![image](https://github.com/user-attachments/assets/3179e7fc-a400-4347-9276-e50662709372)

![image](https://github.com/user-attachments/assets/a11e8a08-af90-45db-b7fb-420c75802307)

**Comment by JoshJuncker** [comment]
Yes. You can have specific conversations appear. Look at the readme for `recordSentMessage` so the operating system knows that there are specific conversations.
```ShareHandlerPlatform.instance.recordSentMessage(
                            conversationIdentifier: "custom-conversation-identifier",
                            conversationName: "John Doe",
                            conversationImageFilePath: path,
                            serviceName: "custom-service-name",
                          );
```

**Comment by JoshJuncker** [comment]
I'm closing this now. But feel free to open it again if there are issues.
